### PR TITLE
TDD project

### PR DESCRIPTION
**Pull request by harshitajain06**
In this project, we:
- Created a class called <code>Solver</code> as described in the [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/testing/tdd_project.md)
- Create a method called <code>factorial</code> as described in the [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/testing/tdd_project.md)
- Create a method called <code>reverse</code> as described in the [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/testing/tdd_project.md)
- Create a method called <code>fizzbuzz</code> as described in the [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/testing/tdd_project.md)

### CODE REVIEW

File: spec/solver_spec.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      expect(solve.fizzbuzz(5)).to eq('buzz')
+    end
+  end
+
+  describe '#fizzbuzz' do
+    it 'returns fizz' do
+      solve = Solver.new
+      expect(solve.fizzbuzz(3)).to eq('fizz')
+    end
+  end
```
Comment (muneebulrehman) [inline spec/solver_spec.rb:44]: Kindly write a test for a number that returns neither `fizz `, `buzz` nor `fizzbuzz`. Eg., number **7** and it should return the number itself.

File: README.md [hunk 2]
```diff
@@ -36,192 +8,89 @@ After you're finished please remove all the comments and instructions!
...
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FUTURE FEATURES -->
-
-## 🔭 Future Features <a name="future-features"></a>
```
Comment (muneebulrehman) [inline README.md:218]: Kindly include a section **Future features** with 1-3 potential features to be implemented in the future.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @harshitajain06 and @DJ-MrJay 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - All changes requested have been implemented  ✅ 
 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**